<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LovdUploadDialog</class>
 <widget class="QDialog" name="LovdUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOVD variant publication</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample data:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Processed sample:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="processed_sample">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="gender">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>n/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Variant data:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="8" column="2">
      <widget class="QComboBox" name="classification2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>HGVS.g:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>HGVS.p:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="hgvs_p2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>HGVS.c:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="hgvs_c"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="hgvs_c2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gene:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Transcript:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="classification">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="genotype">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>het</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="chr">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chr22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chrX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chrY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chrM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chr:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Genotype:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Classification:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="gene"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="hgvs_p"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nm_number"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="hgvs_g">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refseq_btn">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="GSvar.qrc">
           <normaloff>:/Icons/WebService.png</normaloff>:/Icons/WebService.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="hgvs_g2">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refseq_btn2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="GSvar.qrc">
           <normaloff>:/Icons/WebService.png</normaloff>:/Icons/WebService.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="genotype2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>het</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Phenotype data:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PhenotypeSelectionWidget" name="phenos" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upload_btn">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Upload result:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="comment_upload">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="print_btn">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhenotypeSelectionWidget</class>
   <extends>QWidget</extends>
   <header>PhenotypeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>processed_sample</tabstop>
  <tabstop>gender</tabstop>
  <tabstop>chr</tabstop>
  <tabstop>gene</tabstop>
  <tabstop>nm_number</tabstop>
  <tabstop>hgvs_g</tabstop>
  <tabstop>refseq_btn</tabstop>
  <tabstop>hgvs_c</tabstop>
  <tabstop>hgvs_p</tabstop>
  <tabstop>genotype</tabstop>
  <tabstop>classification</tabstop>
  <tabstop>hgvs_g2</tabstop>
  <tabstop>refseq_btn2</tabstop>
  <tabstop>hgvs_c2</tabstop>
  <tabstop>hgvs_p2</tabstop>
  <tabstop>genotype2</tabstop>
  <tabstop>classification2</tabstop>
  <tabstop>upload_btn</tabstop>
  <tabstop>comment_upload</tabstop>
  <tabstop>print_btn</tabstop>
 </tabstops>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections/>
</ui>
